tasks/pkg-redhat.yml

```diff
@@ -3,7 +3,7 @@
   get_url:
     url: "http://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public"
     dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
-    sha256sum: 694a2ffecff85326cc08e5f1a619937999a5913171e42f166e13ec802c812085
+    checksum: sha1:19ae0b5ef027560202632581ba213702cf78be0f
```

Comment:
could we make the algorithm depend on the platform so that we still use a `sha256` hash for platforms that support it?